### DISCUSSION TITLE
Is it possible to mock a response if it meet the rules if not it will go to the real service?

### DISCUSSION THREAD

**Discussion by anton578** [mockoon/mockoon#1101] [Help]
Right now I have built the route with a rules if the route params is of certain value it will respond to a mock data. otherwise it will go to the real service, but right now it is always mocking the data regardless of the value of the route param.

**Comment by 255kb** [comment]
Not yet, but someone is working on it! See https://github.com/mockoon/mockoon/pull/1068
